### DISCUSSION TITLE
Is blitz custom server support removed?

### DISCUSSION THREAD

**Discussion by sharmapukar217** [blitz-js/blitz#3938] [General]
I followed the documentation exactly same but when i use blitz dev, it shows blitz/custom-server is not found. Am i missing something?

**Comment by dartenie** [comment]
You do it the same way as a next custom server. You will also be unable to start the server using "blitz dev" and will instead have to compile/run it however you please. ts-node usually works. You can see a minimal setup for express here (only 3 files must be changed/added):
https://github.com/Pridestalkerr/blitz-custom-server
